README.md

```diff
@@ -10,7 +10,39 @@ This is a helper package that automatically installs and updates the [Angular La
 * Restart Sublime.
 
 
-
 ## Configuration
 
 Open configuration file using command palette with `Preferences: LSP-angular Settings` command or opening it from the Sublime menu (`Preferences > Package Settings > LSP > Servers > LSP-angular`).
+
+
+## Different Angular Version
+
+For people who need an older version of the angular server,
+this is how to point to a local @angular/language-server.
+
+Override the start command in a ST project file, to point to a local @angular/language-server
+
+```json
+{
+  "folders": [
+    {
+      "path": "."
+    }
+  ],
+  "settings": {
+    "LSP": {
+      "LSP-angular": {
+        "command": [
+          "${node_bin}",
+          "${folder}/node_modules/@angular/language-server/index.js", // maybe adjust the path based on your project
+          "--ngProbeLocations", "${folder}/node_modules",  // maybe adjust the path based on your project
+          "--tsProbeLocations", "${folder}/node_modules",  // maybe adjust the path based on your project
+          "--stdio"
+        ]
+      },
+    }
+  }
+}
+```
+
+An example can be found at https://github.com/sublimelsp/LSP-angular/pull/97#issuecomment-1571887048
```

Comment:
I would suggest removing this sentence, 
because you already provided one example in the README above.

That example alone should be clear to the user. 
It would be better to not force the user to click an additional link if the user don't has to.